### DISCUSSION TITLE
Some questions about the color definitions

### DISCUSSION THREAD

**Discussion by nielsnl68** [moononournation/Arduino_GFX#252] [Q&A] [answered]
Hi,
Some time ago i opted an PR where the pre defined *colors* are disabled by a new define.
Sadly after that i found that they are being used in other parts of the library as well. In my fork i just removed those parts for now.
But it is not an nice solution at all. 
One nice solution is to change the color names in your library from `RED` etc. to `GFX_RED` like tft_espi does with `TFT_RED` etc.

thanks anyway.

**Comment by moononournation** [comment]
What is the actual problem you are facing now?

**Comment by nielsnl68** [comment]
sorry for the delay, i started 3 times already in the last days but got sidetracked and lost the parts i wrote down.

The issue i face is that esphome and Arduino_GFX using the same terms (defines) but using both different values Arduino_GFX uses `uint16_t` and esphome has uses a special Color object for the colors.

**Comment by moononournation** [comment, marked as answer]
Please get update and check is it ok. If you defined DISABLE_COLOR_DEFINES, you still can access color by the define with prefix RGB565_, e.g. RGB565_RED

**Comment by nielsnl68** [comment]
Thanks @moononournation, i hope that i can work on it very soon again. I needed some changes included in esphome as well to be merge as well.
